README.md

```diff
@@ -105,7 +105,7 @@ The compiled binary file is in the build directory
 
 ```
 > ls build
-ts-meta  ts-server  ts-sql  ts-store
+ts-cli  ts-meta  ts-server  ts-sql  ts-store

```

Comment:
"git show" found an extra ^M at the end of the line, please check your git config.
![image](https://user-images.githubusercontent.com/4780969/198338575-60f8ddcc-4b22-4a34-adbe-1392b7c575b5.png)

Reply:
the line break of README.md is already CRLF, we can change it in a new issue.